specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/elasticsan.json

```diff
@@ -0,0 +1,2229 @@
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "VolumeGroupList": {
+      "type": "object",
+      "description": "List of Volume Groups",
+      "additionalProperties": false,
+      "properties": {
+        "value": {
+          "description": "An array of Volume Groups objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VolumeGroup"
+          }
+        },
+        "nextLink": {
+          "description": "URI to fetch the next section of the paginated response.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "VolumeGroupProperties": {
+      "type": "object",
+      "description": "VolumeGroup response properties.",
+      "properties": {
+        "provisioningState": {
+          "description": "State of the operation on the resource.",
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "protocolType": {
+          "description": "Type of storage target",
+          "$ref": "#/definitions/StorageTargetType"
+        },
+        "encryption": {
+          "description": "Type of encryption",
+          "$ref": "#/definitions/EncryptionType"
+        },
+        "networkAcls": {
+          "$ref": "#/definitions/NetworkRuleSet",
+          "description": "A collection of rules governing the accessibility from specific network locations.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "privateEndpointConnections": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          },
+          "description": "The list of Private Endpoint Connections."
+        }
+      }
+    },
+    "VolumeGroupUpdate": {
+      "type": "object",
+      "description": "Volume Group request.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties of VolumeGroup.",
+          "$ref": "#/definitions/VolumeGroupUpdateProperties"
+        }
+      }
+    },
+    "VolumeGroupUpdateProperties": {
+      "type": "object",
+      "description": "VolumeGroup response properties.",
+      "properties": {
+        "protocolType": {
+          "description": "Type of storage target",
+          "$ref": "#/definitions/StorageTargetType"
+        },
+        "encryption": {
+          "description": "Type of encryption",
+          "$ref": "#/definitions/EncryptionType"
+        },
+        "networkAcls": {
+          "$ref": "#/definitions/NetworkRuleSet",
+          "description": "A collection of rules governing the accessibility from specific network locations.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        }
+      }
+    },
+    "VolumeCreateParameter": {
```

Comment:
>VolumeCreateParameter [](http://example.com/codeflow?start=5&length=21)

Looks like you don't need this any more either. Unifying GET and PUT definitions was an excellent change.

Reply:
Thanks, i totally forgot to remove it.